cognite/neat/workflows/steps/step_model.py

```diff
@@ -7,6 +8,16 @@
 from cognite.neat.workflows.model import WorkflowConfigItem, WorkflowConfigs
 
 
+class StepCategory(StrEnum):
```

Comment:
Same comment as before, suggest getting the category from the module name instead to avoid redundant informration.

Reply:
This is unused, forgot to remove it.